Controller/CRUDController.php

```diff
@@ -1364,4 +1363,18 @@ protected function preList(Request $request)
 
         return $this->get('translator')->trans($id, $parameters, $domain, $locale);
     }
+
+    protected function setFormTheme(FormView $formView)
```

Comment:
Would prefer a private method, because we will move the twig stuff to the template completly.